client/src/main/java/io/pravega/client/netty/impl/FlowHandler.java

```diff
@@ -194,6 +206,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
             try {
                 processor.process(cmd);
             } catch (Exception e) {
+                log.error("ReplyProcessor.process failed for reply {} due to {}", cmd, e.getMessage());
```

Comment:
Same consideration here: for some expected exceptions logging the "message" may be enough (in this case, maybe `error` would be too severe), but for other unexpected exceptions it would be also useful to have the stack trace.

Reply:
Will change the log to warn level and since `processor.processingFailure(e)` is invoked next the stack trace will be logged there for unexpected exception.